kratos/tests/test_restart.py

```diff
@@ -210,9 +220,9 @@ def __execute_restart_utility_load(self, model_part_name, restart_time):
 
         rest_utility = restart_utility.RestartUtility(loaded_model_part, restart_parameters)
 
-        rest_utility.LoadRestart()
+        rest_utility.LoadRestart() #TODO: it would be best to return the loaded_modelpart from this... 
```

Comment:
i do think it would be better to return the reference to the loaded modelpart

the point is that in python you never pass by reference. so if for any reason i was creating a new loaded_model_part in here (which would be legitimiate) then the test would fail.

in any case i preserved compatibility with the current behaviour

Reply:
you mean inside the `__execute_restart_utility_load` function?

In principle I would also be fine with what you propose, but then it gets complicated in the solvers:
the ModelPart used by the solvers is usually created way before it is filled (either from mdpa or restart file)
=> the ModelPart is passed into a function (e.g. the RestartUtility) and then filled, I don't know how returning it would work